cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -241,10 +241,10 @@ public void getMapReduceInfo(HttpRequest request, HttpResponder responder,
     MRJobInfo mrJobInfo = mrJobInfoFetcher.getMRJobInfo(run.toId());
 
     mrJobInfo.setState(runRecordMeta.getStatus().name());
-    // Multiple runTs / endTs by 1000, to be consistent with Task-level start/stop times returned by JobClient
+    // Multiple startTs / endTs by 1000, to be consistent with Task-level start/stop times returned by JobClient
     // in milliseconds. RunRecord returns seconds value.
-    // The start time of the MRJob is when the run record has been marked as RUNNING
-    mrJobInfo.setStartTime(TimeUnit.SECONDS.toMillis(runRecordMeta.getRunTs()));
+    // The start time of the MRJob is when the run record has been marked as STARTED
+    mrJobInfo.setStartTime(TimeUnit.SECONDS.toMillis(runRecordMeta.getStartTs()));
```

Comment:
I think you can also remove the `@NULLABLE` from the parameter of `MRJobInfo#setStartTime`.